packages/vscode-extension/src/handlers.ts

```diff
@@ -908,6 +910,38 @@ async function openMarkdownHandler() {
   }
 }
 
+async function postUpgrade(): Promise<void> {
+  await openUpgradeChangeLogsHandler();
+  await popupAfterUpgrade();
+}
+
+async function popupAfterUpgrade(): Promise<void> {
+  const aadClientSecretFlag = "NeedToSetAADClientSecretEnv";
+  const aadClientSecret = globalStateGet(aadClientSecretFlag, "");
+  if (
+    aadClientSecret !== "" &&
+    workspace.workspaceFolders &&
+    workspace.workspaceFolders.length > 0
+  ) {
+    try {
+      const learnMoreLink = "https://aka.ms/teamsfx-existing-aad-doc";
+      const learnMoreText = "Learn More";
```

Comment:
For extension, we manage all strings in Strings.json file for potential localization.

Reply:
ok

Reply:
fixed